pkg/hubble/relay/relayoption/defaults.go

```diff
@@ -24,11 +24,13 @@ import (
 
 // Default is the reference point for default values.
 var Default = Options{
-	HubbleTarget:       "unix://" + defaults.HubbleSockPath,
-	DialTimeout:        5 * time.Second,
-	RetryTimeout:       30 * time.Second,
-	ListenAddress:      fmt.Sprintf(":%d", hubbledefaults.RelayPort),
-	Debug:              false,
-	BufferMaxLen:       100,
-	BufferDrainTimeout: 1 * time.Second,
+	HubbleTarget:  "unix://" + defaults.HubbleSockPath,
+	DialTimeout:   5 * time.Second,
+	RetryTimeout:  30 * time.Second,
+	ListenAddress: fmt.Sprintf(":%d", hubbledefaults.RelayPort),
+	Debug:         false,
+
+	BufferMaxLen:           100,
+	BufferDrainTimeout:     1 * time.Second,
+	ErrorAggregationWindow: 3 * time.Second,
```

Comment:
Was this value empirically chosen? It seems a little short to me.

Reply:
For small cluster sizes, 3 seconds seems to work nicely: It's long enough such that each node will have a chance to report their error, but short enough so the clients don't have too long for errors to be reported.

But I actually don't know what a good window size would be for larger clusters. Too short and we end up producing unnecessary traffic, too long and we risk merging uncorrelated errors (and clients have to wait long for errors as well).

Reply:
Bumped to 10sec